docs/modules/ROOT/partials/exoscale/delete-nodes.adoc

```diff
@@ -0,0 +1,60 @@
+. Drain the nodes to replace
```

Comment:
Is this partial a duplicate of `docs/modules/ROOT/partials/exoscale/delete-node.adoc`?

Reply:
yes it is an exact duplicate. deleted